src/webgl/camera.js

```diff
@@ -89,7 +89,7 @@ p5.prototype.perspective = function(fovy,aspect,near,far) {
   );
   this._renderer.uPMatrix = p5.Matrix.identity();
   this._renderer.uPMatrix.perspective(fovy,aspect,near,far);
-  this._renderer._isSetCamera = true;
+  this._renderer._setCamera = 'custom';
```

Comment:
to me, the name setCamera seems like a setter function but in reality it's just a property of the renderer class.  Consider renaming to _camera, or _cam